app/policies/item_policy.rb

```diff
@@ -8,7 +8,7 @@ def resolve
   end
 
   def update?
-    scope.exists?(id: item.id)
+    (item.user == @user) || !!(@user.spouse && (item.user == @user.spouse))
```

Comment:
I think that this is a pretty good catch by Prosopite, and this is one of the factors that makes me see it as better than `bullet`.